docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -254,6 +254,10 @@ children:
     url: Serving-static-files.html
     output: 'web, pdf'
 
+  - title: 'Dynamic models, repositories, and controllers'
```

Comment:
maybe `Dynamically adding models, repositories, and controllers`?  to match the title of the docs page.